### PR TITLE
Fluentd support

### PR DESCRIPTION
**Pull request by christianberg**
Add a Symlinker agent, which creates symlinks with pod metadata encoded in the directory structure. Metadata can then be extracted from the filenames by a log shipping agent such as Fluentd.

### CODE REVIEW

**Comment by christianberg** [comment]
👍

File: kube_log_watcher/agents/__init__.py [hunk 1]
```diff
@@ -2,10 +2,12 @@
...
 from kube_log_watcher.agents.appdynamics import AppDynamicsAgent
 from kube_log_watcher.agents.scalyr import ScalyrAgent
+from kube_log_watcher.agents.symlinker import SymlinkerLoader
 
 
 __all__ = (
     AppDynamicsAgent,
     BaseWatcher,
     ScalyrAgent,
+    SymlinkerLoader
```
Comment (mohabusama) [inline kube_log_watcher/agents/__init__.py]: Just styling: adding `,` at the end of the last element in the dict 😉

Comment (christianberg) [reply kube_log_watcher/agents/__init__.py]: 👍  I will merge this and add the comma with the next PR (which will bump the version).

**Comment by mohabusama** [comment]
👍

**Pull request merged**